src/plugins/network/network.c

```diff
@@ -50,6 +50,70 @@ int elektraNetworkAddrInfo (Key * toCheck)
 	return 0;
 }
 
+int elektraPortInfo(Key * toCheck, Key * parentKey) {
+	const Key * meta = keyGetMeta (toCheck, "check/port");
+	const Key * listenMeta = keyGetMeta (toCheck, "check/port/listen");
+	if (!meta && !listenMeta) return 0; /* No check to do */
+	char *endptr = NULL;
+	long portNumber = strtol(keyString(toCheck), &endptr, 10);
+	int portNumberNetworkByteOrder;
+
+	if (*endptr == '\0') {
+		if (portNumber < 0 || portNumber > 65535 || *endptr != 0) {
```

Comment:
I think you can remove the check `*endptr != 0` here. As far as I can tell, `*endptr` should always be `0` in this part of the code, since the if-statement one line above:

```c
if (*endptr == '\0')
```

makes sure that this is the case.